src/fsharp/LexFilter.fs

```diff
@@ -1626,13 +1720,13 @@ type LexFilterImpl (lightStatus: LightSyntaxStatus, compilingFsLib, lexer, lexbu
         // ...
 
         | _, (CtxtInterfaceHead offsidePos :: _) 
-                when isSemiSemi || (if isInterfaceContinuator token then tokenStartCol + 1 else tokenStartCol) <= offsidePos.Column -> 
+                when isSemiSemi || (if relaxWhitespace2OffsideRule || isInterfaceContinuator token then tokenStartCol + 1 else tokenStartCol) <= offsidePos.Column -> 
```

Comment:
There are a whole lot of these `tokenStartCol + 1` changes - what are these changes doing?

Reply:
To make the rules as stated in the XML docs for `relaxWhitespace2OffsideRule` work

Reply:
I need more information here please, plus an example of what does and doesn't pass with this change.  We have to assess more or less every change in this file carefully - there is considerable churn and that may cause regressions and bugs, which would be a shame.

Reply:
OK, I've read the comment for `relaxWhitespace2OffsideRule`, it's much clearer now, thank you